mage_integrations/mage_integrations/sources/google_ads/templates/config.json

```diff
@@ -0,0 +1,13 @@
+{
+    "developer_token": "",
+    "oauth_client_id": "",
+    "oauth_client_secret": "",
+    "refresh_token": "",
+    "start_date": "2020-01-01T00:00:00Z",
+    "login_customer_ids": [
+        {
+            "customerId": "",
+            "loginCustomerId": null
+        }
+    ]
+}
```

Comment:
nit: add empty line at the end of file

Reply:
Fixed on commit bd83786638ad42f1c9a6ce0cef1916c37fe312ce